### PR TITLE
Add: pr template

### PR DESCRIPTION
**Pull request by teesloane**
I don't have to write anything here because there is no template yet.

😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉 😉

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Describe your changes
+
+<!-- Please add a link to a ticket if relevant: eg: "closes #340" -->
+
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added tests.
+- [ ] Do we need to implement analytics?
```
Comment (mveytsman) [inline .github/pull_request_template.md:10]: What does this mean?

Comment (teesloane) [reply .github/pull_request_template.md:10]: I donno, it came from a template 🤷 

Mostly, I think of it as—is there tracking that needs to be implemented alongside the business logic?

Comment (mveytsman) [reply .github/pull_request_template.md:10]: I think it's maybe overkill - otherwise ship it?

Comment (sereprz) [reply .github/pull_request_template.md:10]: I think it's a bit open to interpretation so maybe not super helpful

**Review by sereprz — APPROVED** [review]
LGTM

**Pull request merged**